quint/src/docs.ts

```diff
@@ -50,6 +50,29 @@ export function produceDocs(quintModule: QuintModule): Map<string, Documentation
   return new Map<string, DocumentationEntry>(compact(entries))
 }
 
+/**
+ * Produces a documentation map for the modules' definitions
+ *
+ * @param quintModule the module for which documentation should be produced
+ * @returns a map of definition ids to their documentation
+ */
+export function produceDocsById(quintModule: QuintModule): Map<bigint, DocumentationEntry> {
```

Comment:
This is a copy from the function above, but with the result indexed by id instead of name. We should probably stick with only this one in the future.